tests/tensor/rewriting/test_linalg.py

```diff
@@ -776,11 +777,11 @@ def test_diag_kronecker_rewrite():
     )
 
 
-def test_slogdet_kronecker_rewrite():
+def test_det_kronecker_rewrite():
```

Comment:
Pre-existing tests shouldn't be changed, that way we know we are not losing functionality

Reply:
this test was removed because i removed the slogdet kronecker rewrite. 
ideally we dont need the slogdet rewrites anymore

Reply:
But the rewrite should still work, so the test can stay. A sign of a good refactor is that you don't need to change or delete old tests.

If you remove this I am not confident the refactor was successful